docs/_docs/installation.md

```diff
@@ -4,42 +4,51 @@ description: Official guide to install Jekyll on macOS, GNU/Linux or Windows.
 permalink: /docs/installation/
 ---
 
-Installing Jekyll should be straight-forward if all requirements are met.
-
+- [Requirements](#requirements)
 - [Install on macOS](#macOS)
-- [Install on GNU/Linux](#Linux)
 - [Install on Windows](../windows/)
+- [Upgrade Jekyll](#upgrade-jekyll)
+
+Installing Jekyll should be straight-forward if all requirements are met.
```

Comment:
"...if your system meets the requirements"

Consistent with details under the Requirements heading.